portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,28 @@
   width: 650px;
 }
 
+.w3-cinzel {
+  font-family: 'Cinzel', serif;
+  color: rgb(26, 186, 235);
+  
+}
+
+.w3-cinzel-body {
+  font-family: 'Cinzel', serif;
+  color: rgb(17, 162, 167);
+  
+}
+
+.w3-lora {
+  font-family: 'Lora', serif;
+  color: whitesmoke;
+  
+}
+
+.img-me{}
```

Comment:
no need to specify if you aren't styling

Reply:
Done